### DISCUSSION TITLE
Hourly rates per user depending on Project possible

### DISCUSSION THREAD

**Discussion by xstable** [kimai/kimai#4331] [Questions & Answers] [answered]
Is it possible to have different hourly rates for different projects?

## Example

 I have two developers (A, B) and two projects (X, Y)

Now I wan't to archive this: 

| Developer | Project | rate |
|----|----|----|
| Dev A | Proj X | 50 |
| Dev A | Proj Y | 60 |
| Dev B | Proj X | 55 |
| Dev B | Proj Y | 70 |

I've found that I can set Price & Internal Price in the User-Settings. But is there a chance, to overwrite this for single projects?

## Why it is use full
It's a general case, that the hourly rate differ from project to project, as you have different contracts. 
In those cases both can happen: 
1. All developers in the Project has the same hourly rate (e.g. if Agency agree with the customer to do the work for X EUR / h, and customer don't care, who work on the project)
2. Every developer has it's own hourly rate at the project (e.g. if Agency only bring DEV in a Project, but add a little amount to their hourly rates.. means hourly rate of each developer differ)

**Comment by kevinpapst** [comment, marked as answer]
Read the docs...
https://www.kimai.org/documentation/rates.html
https://www.kimai.org/documentation/projects.html
